generators/app/templates/generators/_index.js.ejs

```diff
@@ -76,8 +92,20 @@ module.exports = class extends LanguagesGenerator {
     }
 
     get writing() {
```

Comment:
I think you should fix #19 at the same time, by adding all methods that can exist in a blueprint generator:
https://github.com/jhipster/generator-jhipster/blob/master/generators/generator-base-blueprint.js

I think only `install` is missing.